plugins/query-resource-limit-group/src/test/java/org/opensearch/plugin/resource_limit_group/CreateResourceLimitGroupRequestTests.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.resource_limit_group;
+
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.io.IOException;
+
+import static org.opensearch.plugin.resource_limit_group.ResourceLimitGroupTestUtils.compareResourceLimits;
+import static org.opensearch.plugin.resource_limit_group.ResourceLimitGroupTestUtils.resourceLimitGroupOne;
+
+public class CreateResourceLimitGroupRequestTests extends OpenSearchTestCase {
```

Comment:
We should use `AbstractSerializingTestCase` which provides the ser/der out of the box and needs minimal effort.